src/DiagramGenerator/Diagram/Board.php

```diff
@@ -216,7 +209,7 @@ public function drawBorder()
      */
     public function draw()
     {
-        $this->image->setImageFormat('png');
+        $this->image->setImageFormat(Texture::IMAGE_FORMAT_PNG);
```

Comment:
My preference is for PRs to do complete jobs — it feels like this PR does half a thing, and the next will do one and a half things. If you want to stop this PR here, that's fine; then please split the next one into two, so that preserving the JPEG is done separately from compressing the PNG.